### PR TITLE
Button add

### PR DESCRIPTION
**Pull request by mKleinCreative**
Check it out ?

### CODE REVIEW

**Comment by Compsight** [comment]
LGTM

File: index.html [hunk 1]
```diff
@@ -41,16 +58,22 @@ <h3 class="panel-title">Teenagers Houses</h3>
             <h3 class="panel-title">Store</h3>
           </div>
           <div class="panel-body">
-            [ store content ]
-          </div>
```
Comment (tannerwelsh) [inline index.html:45]: Whoops! You deleted the closing `</div>` tags here. Watch out for that!

File: index.html [hunk 2]
```diff
@@ -68,14 +101,26 @@ <h3 class="panel-title">Kettles</h3>
...
+              </div>
+            </div>
           </div>
         </div>
 
         <button id="teenagers" class="btn btn-block store-button" type="button">Test Button Block</button>
 
         Points: <span id="points" class="badge">0</span>
       </div>
+       </div>
```
Comment (tannerwelsh) [inline index.html:122]: Hmm, this looks off. Why are these here?

File: index.html [hunk 3]
```diff
@@ -41,16 +58,22 @@ <h3 class="panel-title">Teenagers Houses</h3>
...
-          </div>
-        </div>
-
         <div class="panel panel-default">
           <div class="panel-heading">
             <h3 class="panel-title">Teenagers</h3>
           </div>
           <div class="panel-body">
-            [ content ]
+            <div class="btn-group btn-group-justified" role="group" aria-label="...">
```
Comment (tannerwelsh) [inline index.html:66]: What's the `aria-label` for?

File: js/counter.js [hunk 4]
```diff
@@ -1,16 +1,97 @@
...
+  theaters: 0
 }
 
 function earnPoints() {
   PLAYER.points += 1
   updateUI()
 }
 
+function buyTeenagers(num) {
+  teenagerIncrease = () => {
```
Comment (tannerwelsh) [inline js/counter.js:14]: Returning functions from a function! Nice. 🤓 

Only problem here is that the variable has been defined in the global scope because there's no `var` or `let` or `const` declaration keyword. (See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#Implicit_globals_and_outer_function_scope)

Comment (tannerwelsh) [reply js/counter.js:14]: To fix this, make the function returned a constant, e.g.:

``` js
const teeangerIncrease = ()  => {
```

File: js/counter.js [hunk 5]
```diff
@@ -1,16 +1,97 @@
...
+  return kettleIncrease
+}
+
+function buyTheaters(num) {
+  theaterIncrease = () => {
+    PLAYER.theaters += num
+    updateTheaterLocations()
+  }
+  return theaterIncrease
+}
```
Comment (tannerwelsh) [inline js/counter.js:35]: Since all of these functions do the same thing, they are ripe for some refactoring.

File: js/counter.js [hunk 6]
```diff
@@ -1,16 +1,97 @@
...
+ $('#teenagers').text(PLAYER.teenagers)
+}
+
+function updateKettleStorage() {
+$('#kettles').text(PLAYER.kettles)
+}
+
+function updateTheaterLocations() {
+ $('#theaters').text(PLAYER.theaters)
+}
```
Comment (tannerwelsh) [inline js/counter.js:52]: Same as above - each of these functions has _very similar_ behavior. We could refactor this code into a more powerful `updateUI` function that takes the value/id as a parameter, e.g.:

``` js
function updateUI(component) {
  const selector = '#' + component
  $(selector).text(PLAYER[component])
}
```

File: js/counter.js [hunk 7]
```diff
@@ -1,16 +1,97 @@
...
+  $('#buyTheaters').click(buyTheaters(1));
+})
+
+$(document).ready(function() {
+  $('#buyTenTheaters').click(buyTheaters(10));
+})
+
+$(document).ready(function() {
+  $('#buyHundredTheaters').click(buyTheaters(100));
+})
```
Comment (tannerwelsh) [inline js/counter.js:96]: Instead of writing all these different `$(document).ready(function() { ... })` statements, you can put them all into one, e.g.:

``` js
$(document).ready(function() {
  $('#buyTheaters').click(buyTheaters(1));
  $('#buyTenTheaters').click(buyTheaters(10));
  // ...
})
```

**Review by tannerwelsh — COMMENTED** [review]
Nice additions! Really like the higher-order functions you made to do the incrementing. The code can be cleaned up in a few places as mentioned.

Also, please don't submit files that aren't needed or associated with the feature change (e.g. `.DS_Store` and `npm-debug.log`)
